frontend/interactEM/src/components/parameterupdater.tsx

```diff
@@ -25,15 +25,15 @@ const compareValues = (
   value2: string,
 ): boolean => {
   switch (parameter.type) {
-    case ParameterType.INTEGER:
+    case "int":
       return Number.parseInt(value1, 10) === Number.parseInt(value2, 10)
-    case ParameterType.FLOAT:
+    case "float":
       return Number.parseFloat(value1) === Number.parseFloat(value2)
-    case ParameterType.BOOLEAN:
+    case "bool":
       return value1 === value2
-    case ParameterType.STRING:
-    case ParameterType.STR_ENUM:
-    case ParameterType.MOUNT:
+    case "str":
+    case "str-enum":
+    case "mount":
```

Comment:
This seems like a step backwards? Comparing to string literals vs a set of constants?

Reply:
```ts
export type ParameterType =
  | "str"
  | "int"
  | "float"
  | "bool"
  | "mount"
  | "str-enum"
```